rising/transforms/compose.py

```diff
@@ -227,3 +234,54 @@ def forward(self, *seq_like, **map_like) -> Union[Sequence, Mapping]:
             if rand[idx] > self.dropout[idx]:
                 data = self.transform_call(data, self.transforms[idx])
         return data
+
+
+class OneOf(AbstractTransform):
+    """
+    Apply one of the given transforms.
+    """
+
+    def __init__(self, *transforms: Union[AbstractTransform, Sequence[AbstractTransform]],
+                 weights: Optional[Sequence[float]] = None,
+                 p: float = 1.,
+                 transform_call: Callable[[Any, Callable], Any] = dict_call):
+        """
+        Args:
+            *transforms: transforms to choose from
+            weights: additional weights for transforms
+            p: probability that one transform i applied
+            transform_call: function which determines how transforms are
+                called. By default Mappings and Sequences are unpacked
+                during the transform.
+
+        References:
```

Comment:
Why do we need this reference? It's not like torchio invented this kind of transforms and as far as i can see, we did not copy their code?

Reply:
The code was inspired by their implementation and it was the first i saw something like this there. So I added just in case. We can also remove it if you want

Reply:
I'd remove it. I've seen something similar in albumentations and others... I thing we either need to refer to all of them or None (And I'd prefer None in that case)